core/dive.c

```diff
@@ -2792,6 +2787,9 @@ static bool new_without_trip(const struct dive *a)
  * the dates are *too* far off the user will have to join two
  * dives together manually. But this tries to handle the sane
  * cases.
+ *
+ * Dive a and b are supposed to be the previously existing and a
```

Comment:
This comment is confusing. Do you mean that (a) already exists and (b) is new, or vise versa? Or this could even mean that a and be are already existing. Removing the 'a' from the 'a newly imported dive' might make it more clear, but I would suggest being still more explicit on the comment.